packages/eslint-plugin/README.md

```diff
@@ -39,9 +44,133 @@ Once you've installed the npm package you can enable the plugin and rules by ope
 
 ### @stylexjs/valid-styles
 
-StyleX requires styles that are statically analyzable. This rule will detect invalid styles that stylex cannot handle.
+StyleX requires styles that are statically analyzable. This rule will detect
+invalid styles that stylex cannot handle.
+
+#### Disallowed CSS properties and suggested fixes
+
+This guide lists common CSS properties that are **not allowed** in StyleX, along
+with their **suggested replacements**. The goal is to encourage explicit,
+modular, and logical CSS, with an explicit preference for longhand values.
+
+##### Disallowed: Logical styles for `margin` and `padding`. Note: this is autofixable.
```

Comment:
This reads like it disallows logical direction CSS. But it's also confusing because it mentions `margin` which is a shorthand not a logical property